launchable/commands/subset.py

```diff
@@ -155,6 +156,13 @@
     help="Prioritize tests that failed within the specified hours; maximum 720 hours (= 24 hours * 30 days)",
     type=click.IntRange(min=0, max=24 * 30),
 )
+@click.option(
+    "--prioritized-tests-mapping",
+    "prioritized_tests_mapping",
```

Comment:
In my opinion, we should add `_file` suffix to this variable name

Reply:
[297369c](https://github.com/launchableinc/cli/pull/622/commits/297369c1bd3b6c79ea687c9ad1edc2c285c1a466)